Geometry/HGCalCommonData/data/hgcalConsData/v14/hgcalConsData.xml

```diff
@@ -0,0 +1,4751 @@
+<?xml version="1.0"?>
```

Comment:
@bsunanda this is a rather large file. Is there any way the information in this file can be encoded more concisely? (I'm mainly concerned that if every new HGCal geometry requires a similar file, it will become a non-trivial portion of the entire repository.)

Reply:
Shall try in future. I shall retire V13 file once this is integrated. But that needs changes in the scenario declaration. I shall try to reduce the size in future releases.